code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -150,18 +150,18 @@
 	catalysts = list("phoron" = 1)
 	result_amount = 2
 
-/datum/chemical_reaction/tramadol
-	name = "Tramadol"
-	id = "tramadol"
-	result = "tramadol"
-	required_reagents = list("inaprovaline" = 1, "ethanol" = 1, "oxygen" = 1)
-	result_amount = 3
-
 /datum/chemical_reaction/paracetamol
 	name = "Paracetamol"
 	id = "paracetamol"
 	result = "paracetamol"
-	required_reagents = list("tramadol" = 1, "sugar" = 1, "water" = 1)
+	required_reagents = list("inaprovaline" = 1, "nitrogen" = 1, "water" = 1)
+	result_amount = 2
+
+/datum/chemical_reaction/tramadol
+	name = "Tramadol"
+	id = "tramadol"
+	result = "tramadol"
+	required_reagents = list("paracetamol" = 1, "ethanol" = 1, "oxygen" = 1)

```

Comment:
These changes make Tramadol probably the most complex non-phoron chem to make, since you would have to make it in three steps. (5 carbon, oxygen, sugar > 15 Inaprovaline + nitrogen + water > 30 Paracetamol + ethanol + oxygen > 90 tramadol)

Considering pain is not as central to polaris medical system as it is to brainmed, that seems to be a rather puzzling choice?

Reply:
I feel like it makes sense in the method most other chemicals are made in SS13, with the weaker variant being able to be made into the stronger one. (E.G. kelotane to dermaline, dexalin to dexalin plus, and cryoxadone to clonexadone.)

That's not to say the current method doesn't make sense considering it's basically just tramadol watered down to make a weaker painkiller.

Reply:
You still get 90 Tramadol from the 'optimal' mixing method that exists currently, I've changed it here as there's no reason to make the weaker version if you make Tramadol first, which is much stronger.

Reply:
There's..still no reason to make/use paracetamol. It stops being useful at around 25 damage, which..kinda makes it useless at the start.

Reply:
Inaprovaline always seemed more like standalone stabilizer to me and its use in the painkiller tree confused me.  I agree with paracetamol being an ingridient in the tramadol. But I don't think inaprovaline should be part of that tree.

Reply:
Nah. Inaprov and dylo are the two main "root" chems. Inaprov makes a bit of sense at least, since it also has weak painkilling properties and is in general a better fit than dylovene.

Reply:
The idea is not bad, but I feel that having three steps is a bit much. My personal recommendation is to replace Inaprovaline with a non-compound chem, thus making it two steps.

Reply:
...I feel like this is a bit bikeshedy. Its not that big of a change and won't take much time for people to adjust. I don't see why the change is being made, but I also don't really see why its a point of conflict.